OpenTibia/Assets/Scripts/Modules/Options/LegacyHotkeyOptionsWidget.cs

```diff
@@ -121,10 +121,6 @@ public class LegacyHotkeyOptionsWidget : UI.Legacy.PopUpBase
 
         protected override void OnDisable() {
             base.OnDisable();
-
-            var inputHandler = OpenTibiaUnity.InputHandler;
-            if (inputHandler != null)
-                inputHandler.RemoveKeyUpListener(OnKeyUp);
```

Comment:
Are we completely sure this listener is removed in some point? There is no way it will persist registered despite of the fact there is no need for that?